README.md

```diff
@@ -45,6 +45,7 @@ something more or download the video yourself and pass the path to `skonaki`.
 
 * Dependencies
   * To use `skonaki` you need to install its dependencies first by `pip install --user -r requirements.txt`.
+  * The Whisper OpenAI API is used by default. To instead use Whisper locally, install the Whisper library (including all other dependencies) by `pip install --user -r requirements.txt -r requirements-local-whisper.txt`.
```

Comment:
Wait, isn't it enough to run `pip install --user -r requirements-local-whisper.txt`?
If not, what was the point of referring to the `requirements.txt` from within `requirements-local-whisper.txt`?

Reply:
Yes, I made a typo here (the requirements-local-whisper is the only one necessary to call)! Will fix it now :construction_worker: